templates/humicroedit/subtask-1/templates.yaml

```diff
@@ -0,0 +1,226 @@
+dataset: humicroedit
+subset: subtask-1
+templates:
+  27c7a53c-d5b8-410d-affc-95ff59a89c03: !Template
+    id: 27c7a53c-d5b8-410d-affc-95ff59a89c03
+    jinja: 'Please rate how funny the sentence is from {{"0.0"}} to {{"3.0"}}.
+
+      {{ original.replace(original[original.index("<"):original.index(">")+1], edit)
+      }}
+
+      |||
+
+      {{ ((5 * meanGrade) | round) / 5 }}'
+    name: only_edited_sent_rate
+    reference: ''
+  2d0e8f25-5680-4079-9a59-7b06329bd65a: !Template
+    id: 2d0e8f25-5680-4079-9a59-7b06329bd65a
+    jinja: 'Please rate how funny the sentence after replacing the "{{ original[original.index("<")+1:original.index("/>")]
```

Comment:
This one is grammatically off. Also the examples are difficult to read since the replacement comes after the sentence.